pydicom/tests/test_environment.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2008-2018 pydicom authors. See LICENSE file for details.
+"""Tests for the TravisCI testing environments.
+
+The current pydicom testing environments are as follows:
+
+* conda:
+  * Python 2.7:
+    * no additional packages
+    * numpy
+    * numpy, gdcm
+    * numpy, pillow (jpg, jpg2k)
+    * numpy, jpeg-ls
+    * numpy, pillow (jpg, jpg2k), jpeg-ls
+    * numpy, pillow (jpg, jpg2k), jpeg-ls, gdcm
+  * Python 3.4, 3.5, 3.6, 3.7:
+    * As with 2.7
+  * Python 2.7, 3.7:
+    * numpy, pillow (jpg)
+* pypy
+  * Python 2.7, 3.5:
+    * no additional packages
+    * numpy
+* ubuntu
+  * Python 2.7:
+    * no additional packages
+    * numpy
+
+Environmental variables
+-----------------------
+DISTRIB: conda, pypy, ubuntu
+PYTHON_VERSION: 2.7, 3.4, 3.5, 3.6, 3.7
+NUMPY: true, false
+PILLOW: jpeg, both, false
+JPEG2000: false
```

Comment:
isn't this part of PILLOW?

Reply:
Oh, that's left over from when I was starting work.

Reply:
Removed